3주차/[Lv.3]_야근_지수/주다현.js

```diff
@@ -0,0 +1,18 @@
+function solution(n, works) {
+  if (works.reduce((a, b) => a + b, 0) < n) return 0;
+
+  const sortedArr = works.sort((a, b) => b - a);
+
+  while (n) {
+    let max = sortedArr[0];
```

Comment:
```suggestion
    const max = sortedArr[0];
```

`max` 변수는 하위 스코프에서 더 이상 변경이 발생하지 않는 변수이기 때문에 `const` 로 선언해도 좋을 것 같습니다!!